lib/streamlit/elements/widgets/chat.py

```diff
@@ -281,6 +281,7 @@ def chat_input(
         id = compute_widget_id(
             "chat_input",
             user_key=key,
+            key=key,
```

Comment:
This was done for consistency since both `key` and `user_key` are passed to all other calls to `compute_widget_id`.

Alternatively, I think it'd be safe to remove the `key` kwarg from this call and all the others since the value duplicates that of `user_key`.

Reply:
Was there an initial reason for having `user_key` and `key` here?

Reply:
`user_key` is used by the `compute_widget_id` function so that keys can be appended to the end of widget IDs instead of only being included as part of the hash. I think this is just for added human-readability so that the widget ID of a widget with a user-set key can be easily identified.

I'm not sure the reason why we both have `key` and `user_key` passed to each call to `compute_widget_id`, but that's how things are currently written, so just changing `st.chat` to comply with the current pattern made it easier to implement tests.

We can probably drop one of the two later if I'm right that we don't need both, but I didn't want to make the change without @AnOctopus' input, so we can do so in a quick followup next week if there's no issue with the change.